src/ios/CDVInAppBrowser.m

```diff
@@ -246,25 +246,11 @@ - (void)show:(CDVInvokedUrlCommand*)command
 - (void)hide:(CDVInvokedUrlCommand*)command
```

Comment:
Just make it 
```- (void)hide:(CDVInvokedUrlCommand*)command
{
    [self browserHide];
}````
There is no need to create a new hide method, you can call browserHide directly